mockup/patterns/tinymce/js/links.js

```diff
@@ -466,6 +466,14 @@ define([
         upload: this.options.upload,
         text: this.options.text,
         insertHeading: this.options.text.insertHeading,
+        insertImageHelp: this.options.text.insertImageHelp,
+        ExternalImage: this.options.text.ExternalImage,
```

Comment:
is ``externalImage`` in mockup master

Reply:
there are also other upper/lowercase inconsitencies here.
@djowett-ftw could you please just copy over the whole generateModalHtml function from master? I'd also like to have the attributes in the same order.

Reply:
Hi @thet - good idea I've done that.